src/Raven.Server/Documents/Handlers/AttachmentHandler.cs

```diff
@@ -102,6 +102,7 @@ public Task GetPost()
                         HttpContext.Response.Headers.Remove("Content-Type");
                 }
                 HttpContext.Response.Headers["Content-Hash"] = attachment.Base64Hash.ToString();
+                HttpContext.Response.Headers["Content-Size"] = attachment.Stream.Length.ToString();
```

Comment:
This is problematic. It looks like we meant `Content-Length` but put something else there.
Better to put `Total -Attachment-Size` doesn't look like a known header